task-tracker-console-application/src/main/java/com/isa/MainMenu.java

```diff
@@ -0,0 +1,42 @@
+package com.isa;
+
+import java.util.Scanner;
+
+public class MainMenu {
+    public static void displayMainMenu() {
+        System.out.println("=====================================");
+        System.out.println("         MENU GŁÓWNE");
+        System.out.println("=====================================");
+        System.out.println("Opcje:");
+        System.out.println("1. Rejestracja ");
+        System.out.println("2. Logowanie");
+        System.out.println("3. Zaloguj się jako administrator ");
+        System.out.println("4. Zakończenie programu");
+        System.out.println();
+        System.out.println("Wybierz opcję :");
+    }
+    public static void printUserMainMenuChoice(){
+        Scanner scanner = new Scanner(System.in);
+
+            int userChoice = scanner.nextInt();
+                switch (userChoice){
+                    case 1 :
+                        System.out.println("rejestracja niedostepna");
```

Comment:
dobrze by było, żeby aplikacja po każdym złym kroku wróciła do menu; dodatkowo nie łapiemy tu błędów, a jak wpisze tekst zamiast liczby to mi poleci InputMismatchException; można to albo zrobić tu, albo w osobnym tasku

Reply:
@malik-maciej Mamy do tego sobnego taska [JJDZR12MF-9](https://jira.is-academy.pl/browse/JJDZR12MF-9)